setup.py

```diff
@@ -28,7 +28,7 @@ def write_version_to_file(version, target_file):
 
 
 if __name__ == '__main__':
-    version = '0.6.1+%s' % get_git_commit_number()
+    version = '0.6.2+%s' % get_git_commit_number()
```

Comment:
Do you think this should be a minor or major version bump?

Reply:
I think we can do a major version bump: 0.7.0.